package.json

```
@@ -2,7 +2,7 @@
   "name": "spryker-vsf-b2c",
   "description": "B2C project based on VueStorefront Next with Spryker integration",
   "author": "Spryker Systems",
-  "version": "0.0.1",
+  "version": "0.1.1-next.0",
```

Please update to the stable version
```suggestion
  "version": "0.1.1",
```